src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -289,7 +292,7 @@ private void ResetContentViews()
         private void RunOnSchedulerSync(Action a, AsyncTaskCompletedHandler h)
         {
             var task = new DelegateBasedAsyncTask(scheduler, a);
-            task.ThenPost(h, DispatcherSynchronizationContext.Current);
+            task.ThenPost(h, SynchronizationContext.Current);
```

Comment:
`Current` static member belongs `SynchronizationContext`, so resharper suggested this change